### PR TITLE
In Feature Engineering Automation updated automl repo which was showing repo not found

### PR DESCRIPTION
**Pull request by abhi526691**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -400,7 +400,8 @@ www.youtube.com/watch?v=zbjub8BQPyE)
...
 * [AutoGluon](https://autogluon.mxnet.io/) ![](https://img.shields.io/github/stars/awslabs/autogluon.svg?style=social) - Automated feature, model, and hyperparameter selection for tabular, image, and text data on top of popular machine learning libraries (Scikit-Learn, LightGBM, CatBoost, PyTorch, MXNet)
 * [AutoML-GS](https://github.com/minimaxir/automl-gs) ![](https://img.shields.io/github/stars/blue-yonder/tsfresh.svg?style=social) - Automatic feature and model search with code generation in Python, on top of common data science libraries (tensorflow, sklearn, etc)
-* [automl](https://github.com/ClimbsRocks/auto_ml) ![](https://img.shields.io/github/stars/ClimbsRocks/automl.svg?style=social) - Automated feature engineering, feature/model selection, hyperparam. optimisation
+* [automl](https://github.com/ClimbsRocks/auto_ml) ![](https://img.shields.io/github/stars/ClimbsRocks/auto_ml?style=social)  - Automated feature engineering, feature/model selection, hyperparam. optimisation
+
```
Comment (axsaucedo) [inline README.md:404]: Seems there is an extra line here

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Thanks for the correction, seems there is an extra line added

File: README.md (hunk 1 above)
Comment (abhi526691) [reply README.md:404]: @axsaucedo  that is the image shield link which I have added

Comment (abhi526691) [reply README.md:404]: @axsaucedo that is the image shield link which I have added

Comment (axsaucedo) [reply README.md:404]: I mean there's an extra blank line between automl and colombus (bullet point below)

**Comment by axsaucedo** [comment]
Ive edited manually - thanks for the contribution!

**Pull request merged**